src/customers/credit-company-activity.md

```diff
@@ -1,8 +1,6 @@
 ---
 b2b_only: true
 title: Company Credit Activity
-redirect_from:
```

Comment:
Let's keep the existing redirects, which are a result of some previous movement.

```suggestion
redirect_from:
  - /sales/credit-company-activity.html
 ```